include/SANDKalmanFilter.h

```diff
@@ -0,0 +1,217 @@
+#ifndef SANDKALMANFILTER_H
+#define SANDKALMANFILTER_H
+
+#include "TGeoManager.h"
+#include "TMatrixD.h"
+
+#include "SANDTrackerClusterCollection.h"
+#include "SANDKFTrack.h"
+#include "SANDTrackerUtils.h"
+
+struct SParticleInfo {
+  int charge;
+  double mass;
+  int pdg_code;
+  int id;
+
+  TVector3 pos;
```

Comment:
TVector3 is a legacy class.

https://root.cern.ch/doc/master/classTVector3.html